.demo/env/charts/gatekeeper.yaml

```diff
@@ -0,0 +1,75 @@
+charts:
+    gatekeeper:
+        enabled: false
+        constraints:
+            - parameters:
+                enabled: false
+                cpu: 10m
+                memory: 120Mi
+              policyName: containerlimits
+            - parameters:
+                enabled: true
+                tags:
+                    - ''
+                    # - 0.3.7-debian-10-r103
+                    # - 5.0.7-debian-10-r32
+                    # - v1.1.0
+                    # - latest
+                    # - master
+              policyName: bannedimagetags
+            - parameters:
+                enabled: true
+                repos:
+                    - otomi
+                    - mcr.microsoft.com
+                    - k8s.gcr
+                    - eu.gcr.io
+                    - quay.io
+                    - gcr.io
+                    - ghcr.io
+                    - us.gcr.io
+                    - busybox
+                    - docker.io
+                    - goharbor
+                    - grafana
+                    - istio
+                    - drone
+                    - prom
+                    - bats
+                    - kiwigrid
+                    - dduportal
+                    - weaveworks
+                    - curlimages
+                    - kennethreitz
+                    - angelbarrera92
+              policyName: allowedrepos
+            - parameters:
```

Comment:
again, why is this configurable? This needs to be set only to serve as sane defaults. Why would anybody want to change this?

Reply:
hmm, you are saying something contradictory here. 
These are values a policy can accept. It cannot be non-configurable. 

If that is the case, then values need to be coded in the rego file. 

If we want defaults, they are parameters passed as data.

All the policies (what we have and other examples from various libraries Konstraint/Gatekeeper, etc) need to be revisited and decide what parameters should our stack verify for.

For example, the parameters for **allowedrepos** policy is a list of all the currently used container registry repositories. That policy checks if the container has the image coming from one of these and will fail otherwise. This means that for bringing a new image in the stack, the developer also has to update this policy parameters with a new repository, if that is the case. 

Does this policy even make sense? **I don't know, my focus was on building the feature in the stack**. Now we can focus on policies and their parameters.